app/models/search/bookmark_query.rb

```diff
@@ -124,7 +124,7 @@ def default_sort
   end
 
   def sort
-    sort_hash = { sort_column => { order: sort_direction } }
+    sort_hash = { sort_column => { order: sort_direction }, "id" => { order: sort_direction } }
```

Comment:
I'm not sure if I like having both sort columns as keys in the same hash. It means that we're relying on both the ruby end and the elasticsearch end to respect the order of the keys, and while it seems that both ends do respect the order in the current implementation, I'm not sure whether that's guaranteed as part of the official specifications. I'd prefer if this was an array of hashes, since the order there is guaranteed to be correct.